test/OpenTelemetry.Instrumentation.Http.Tests/http-out-test-cases.json

```diff
@@ -90,35 +90,37 @@
   {
     "name": "Call that cannot resolve DNS will be reported as error span",
     "method": "GET",
-    "url": "https://sdlfaldfjalkdfjlkajdflkajlsdjf.sdlkjafsdjfalfadslkf.com/",
+    "url": "http://sdlfaldfjalkdfjlkajdflkajlsdjf:{port}/",
```

Comment:
why changed these?

Reply:
wanted to validate we capture port information as well.